test/system/600-completion.bats

```diff
@@ -309,6 +309,12 @@ function _check_no_suggestions() {
     # recurse for any subcommands.
     check_shell_completion
 
+    # check inspect with format flag
+    run_podman create --name inspect-$random_container_name $IMAGE
+    run_completion inspect inspect-$random_container_name -f \"{{.\"
+    arrOp=(${output//>/ })
+    is "${arrOp[0]}" ":6"
```

Comment:
We should check with something like `assert "$output" =~ ".*^{{.Args}}\$.*" "Container type is completed"`
Checking `:6` is kinda pointless and does not say anything about the actual values.